bindata/assets/alerts/cpu-utilization.yaml

```
@@ -14,11 +14,15 @@ spec:
               cause a cascading failure; increase available CPU.
             runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md
             description: >-
-              Given three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
+              On a multi-node cluster with three control plane nodes, the overall CPU utilization may only be about 2/3 of all available capacity.
               This is because if a single control plane node fails, the remaining two must handle the load of the cluster in order to be HA.
-              If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to
-              fail when they take the load.
+              If the cluster is using more than 2/3 of all capacity, if one control plane node fails, the remaining two are likely to fail when they take the load.
               To fix this, increase the CPU and memory on your control plane nodes.
+              
+              On a SNO cluster, this alert will also fire if the 2/3 of the CPU cores of on the SNO are in use by any workload. This level of CPU utlization
```

(suggestion) define SNO acronym first
```suggestion
              On a single node OpenShift (SNO) cluster, this alert will also fire if the 2/3 of the CPU cores of the node are in use by any workload. This level of CPU utlization
```